pulsar-broker/src/test/java/org/apache/pulsar/broker/service/persistent/PersistentTopicTest.java

```diff
@@ -558,10 +559,10 @@ public void testCreateTopicWithZombieReplicatorCursor(boolean topicLevelPolicy)
         admin.tenants().updateTenant("prop", tenantInfo);
 
         if (topicLevelPolicy) {
-            admin.topics().setReplicationClusters(topicName, Collections.singletonList(remoteCluster));
+            admin.topics().setReplicationClusters(topicName, Arrays.asList(remoteCluster, "test"));
```

Comment:
Why did you add another cluster "test" to this test? Even without the change of `PersistentTopic`, this test could still pass.

Reply:
- If we disabled replication, the `clusters` will contain one cluster `test`. See: https://github.com/apache/pulsar/blob/master/pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BrokerTestBase.java#L55-L56
- If we enabled replication, the `clusters1 will contain two clusters: local cluster and remote cluster

If the cluster is not set correctly, it can mislead others: `clusters` is an empty collection when Replication is disabled.

Reply:
I think it's another topic about the difference determined by whether the replication is enabled. It does not affect this test.

Reply:
> I think it's another topic about the difference determined by whether the replication is enabled. It does not affect this test.

Correct.

Reply:
So, it's better to revert this unrelated change.

Reply:
reverted thse changes